.circleci/config.yml

```diff
@@ -18,31 +17,36 @@ commands:
       - run: echo $(python --version) > .python-version
       - restore_cache:
           key: py-deps-{{ checksum ".python-version" }}-{{ checksum "docs/requirements.txt" }}
-      - run: "if [ ! -d ./venv ]; then python -m venv ./venv; fi"
+      - run: 'if [ ! -d ./venv ]; then python -m venv ./venv; fi'
       - run: echo "source $(pwd)/venv/bin/activate" >> $BASH_ENV
       - run: pip install -r ./docs/requirements.txt
       - save_cache:
           key: py-deps-{{ checksum "docs/requirements.txt" }}
           paths:
             - ./venv
 
-
 aliases:
   - &node12
     image: circleci/node:12
 
-  - &python-docs
-    # using Python 3.6 as that's the version ReadTheDocs is running
+  - &python-docs # using Python 3.6 as that's the version ReadTheDocs is running
     image: circleci/python:3.6-node
 
   - &test-steps
     steps:
       - checkout
       - npm-install
+      - run: npm run build
```

Comment:
Should the build be an explicit step or should it be implicitly included as something like `pretest` script?

Reply:
I've tried to perform a build once and store it, but since its quite fast, we've decided with @kylef that such setup may be an overkill (attaching a workspace with the build takes multiple times longer than the build itself). I'm okay to revisit this, if you find it necessary.